pkgs/build-support/fetchurl/default.nix

```
@@ -8,9 +8,13 @@
   cacert ? null,
   rewriteURL,
   hashedMirrors,
+
+  # Used in `pkgs/build-support/prefer-remote-fetch/default.nix`
+  preferLocalBuild ? true,
 }:
 
 let
+  cpPreferLocalBuild = preferLocalBuild;
```

This should be consistent with the rest of the PR:

```suggestion
  preferLocalBuildDefault = preferLocalBuild;
```